src/feeds/ChainlinkEmaPriceFeed.sol

```diff
@@ -0,0 +1,149 @@
+pragma solidity ^0.8.20;
+
+import "src/util/FeedLib.sol";
+
+// Combined Chainlink EMA Price Feed, allows for additional fallback to be set
+
+contract ChainlinkEmaPriceFeed {
+    IChainlinkFeed public immutable assetToUsd;
+    IChainlinkFeed public assetToUsdFallback;
```

Comment:
Can use `ChainlinkBasePriceFeed` for `assetToUsd` and get rid of the `assetToUsdFallback`